### ISSUE TITLE
[Bug] Minor Memory Leak in CameraView

### ISSUE THREAD

**Issue by tikimcfee** [wonderkiln/CameraKit-Android#186]
https://github.com/wonderkiln/CameraKit-Android/blob/3a88b3a50de1045b2cadfc41778447a42a016836/camerakit/src/main/java/com/wonderkiln/camerakit/CameraView.java#L200

Greetings!

Not sure if this is from your latest code base, but I discovered a nice little memory leak in the CameraView. Turns out, the above lines of code uses the view's context to get an instance of the compat manager object. That method, though, caches off contexts(!) rather unexpectedly:

```
/**
 * Gets an instance of the display manager given the context.
 */
public static DisplayManagerCompat getInstance(Context context) {
    synchronized (sInstances) {
        DisplayManagerCompat instance = sInstances.get(context);
        if (instance == null) {
            final int version = android.os.Build.VERSION.SDK_INT;
            if (version >= 17) {
                instance = new JellybeanMr1Impl(context);
            } else {
                instance = new LegacyImpl(context);
            }
            sInstances.put(context, instance);
        }
        return instance;
    }
}
```

So long story short, if this camera view used in an activity, it will leak the activity itself in the display manager. The fix is super easy - just change `getContext()` to `getContext().getApplicationContext()`.

Thanks much for a great lib - looking forward to this getting better and better!
